src/resty/aws/credentials/ProfileCredentials.lua

```diff
@@ -0,0 +1,41 @@
+--- EnvironmentCredentials class.
+-- @classmod EnvironmentCredentials
+
+
+-- Create class
+local Super = require "resty.aws.credentials.Credentials"
+local config = require "resty.aws.config"
+local ProfileCredentials = setmetatable({}, Super)
+ProfileCredentials.__index = ProfileCredentials
+
+
+--- Constructor, inherits from `Credentials`.
+--
+-- _Note_: this class will fetch the credentials upon instantiation. So it can be
```

Comment:
Is this copied from `EnvironmentCredentials`?
Need to change the content to describe the logic of fetching creds from the creds file, otherwise, the generated document will be wrong